pkg/etcd/etcd.go

```diff
@@ -707,7 +706,7 @@ func getClientConfig(ctx context.Context, control *config.Control, endpoints ...
 	}
 
 	var err error
-	if strings.HasPrefix(endpoints[0], "https://") {
+	if strings.HasPrefix(endpoints[0], "https://") || strings.HasPrefix(endpoints[0], "unix://") {
```

Comment:
should only configure tls for unixs scheme
```suggestion
	if strings.HasPrefix(endpoints[0], "https://") || strings.HasPrefix(endpoints[0], "unixs://") {
```

Reply:
I removed it since we don't actually want to run `etcd only` nodes with kine